src/sentry/integrations/slack/integration.py

```diff
@@ -62,24 +62,45 @@ class SlackIntegrationProvider(IntegrationProvider):
     features = frozenset([IntegrationFeatures.CHAT_UNFURL, IntegrationFeatures.ALERT_RULE])
 
     # Scopes differ depending on if it's a workspace app
-    identity_oauth_scopes = (
-        frozenset(["bot", "links:read", "links:write"])
-        if not settings.SLACK_INTEGRATION_USE_WST
-        else frozenset(
-            [
-                "channels:read",
-                "groups:read",
-                "users:read",
-                "chat:write",
-                "links:read",
-                "links:write",
-                "team:read",
-            ]
-        )
+    wst_oauth_scopes = frozenset(
+        [
+            "channels:read",
+            "groups:read",
+            "users:read",
+            "chat:write",
+            "links:read",
+            "links:write",
+            "team:read",
+        ]
+    )
+
+    # some info here: https://api.slack.com/authentication/quickstart
+    bot_oauth_scopes = frozenset(
+        [
+            "channels:read",
+            "groups:read",
+            "users:read",
```

Comment:
I think we need `im:read` to be able to use the [conversations.list](https://api.slack.com/methods/conversations.list) endpoint down the road.

Reply:
@MeredithAnya Nah, you just need one of the scopes on that list. We have `channels:read` and `groups:read`. I've tested this in Postman as well.

Reply:
I also just tested out using the types `im` in the params and I get the following back:
```
"ok": false,
    "error": "missing_scope",
    "needed": "im:read",
    "provided": "links:write,channels:read,groups:read,chat:write,chat:write.public,chat:write.customize,links:read,team:read,users:read"
```

I think we need that scope if we want to check for users as valid channels for alert rules

Reply:
@MeredithAnya you are right! Good catch